File: pgm/ABC_Java/ABC_404_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String S = reader.readLine();
        reader.close();
        final char a = 'a';
        final char z = 'z';
        final int one = 1;
        for (char c = a; c <= z; c++) {
            if (S.indexOf (c) == -one) {
                System.out.println (c);
                break;
            }
        }
        System.exit (0);
    }
}

File: pgm/ABC_Kotlin/ABC_404_A.kt
// import java.time.*

// import kotlin.math.*

const val a: Char = 'a'
const val z: Char = 'z'

fun main() {
    val S: String = readln()
    for (c in a..z) {
        if (c !in S) {
            println(c)
            break
        }
    }
    kotlin.system.exitProcess(0)
}
